src/nlp_expr.jl

```diff
@@ -1151,3 +1121,16 @@ end
 function LinearAlgebra.qr(::AbstractMatrix{<:AbstractJuMPScalar})
     return throw(MOI.UnsupportedNonlinearOperator(:qr))
 end
+
+# Add support for AbstractArray arguments in GenericNonlinearExpr
```

Comment:
I think for now, we should remove these functions. It's a trivial addition and you could pirate the methods in `DCP.jl` during development.

Reply:
Yes